cmd/tailscaled/tailscaled.go

```diff
@@ -340,6 +341,13 @@ func run() (err error) {
 
 	sys := new(tsd.System)
 
+	healthTracker := sys.HealthTracker()
+	health.MetricHealthMessage.Set(health.MetricHealthMessageLabel{
```

Comment:
This is kind of weird now with the metric being defined in `health`, but set here.
Perhaps just set it in the `health` package as well? E.g. https://github.com/tailscale/tailscale/commit/f12aff959aa40b1085dee5def507273538230d65